packages/server/app/Controllers/Http/AuthenticationController.ts

```diff
@@ -247,6 +248,8 @@ export default class AuthenticationController {
             'resendConfirmationEmail',
         );
         if (hasReachedRateLimit === true) {
+            response.status(429);
```

Comment:
Http status code, just like english irregular verbs should be learned at school !

Reply:
Lol, actually Github Copilot suggested me the status code. 😆